sysutils/tiptop/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           python  1.0
+
+github.setup        nschloe tiptop 0.0.16 v
+github.tarball_from archive
+revision            0
+
+description         \
+    Command-line system monitoring
+
+long_description    \
+    tiptop is a command-line system monitoring tool in the spirit of top. It \
+    displays various interesting system stats, graphs it, and works on Linux \
+    and macOS.
```

Comment:
can this not be just aligned with the rest of the Portfile (I know.. it's a personal preference)

Reply:
Definitely personal preference.  I use to use the same alignment for long blocks of text, but I prefer this style nowadays as it feels more readable (to me).

Reply:
I can see that point for the `long_description` but see no need to have the `description` start on a newline